web/public/css/style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css?family=Dosis|Roboto');
+
+
+/* FU SASS, CSS VARIABLES YEEEE */
+:root {
+  --header-font: 'Dosis', sans-serif;
+  --body-font: 'Roboto', sans-serif;
+}
+
+html {
+
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  height: 100vh;
```

Comment:
Maybe it would be neater to use `min-height` to allow the body to grow past the viewport? This will lead to scrollbars inside boxes if there is an overflow.